### PR TITLE
Connect data to charts

### PR DESCRIPTION
**Pull request by jchen23-stf**
# Connected Survey Charts to actual data

## :recycle: Current situation & Problem
The charts for each of the surveys were displaying dummy data.

## :bulb: Proposed solution
We connected the chart view to actual survey data.

## :gear: Release Notes 
[*Add a summary of the feature and possible migration guides if this is a breaking change so this section can be added to the release notes.*
*Include code snippets that provide examples of the feature implemented if it appends or changes the public interface.*](https://user-images.githubusercontent.com/56773304/225556067-2a01d003-af4f-4b23-838b-5c2f8fbc815e.mp4)

## :heavy_plus_sign: Additional Information
*Provide some additional information if possible*

### Related PRs
N/A

### Testing
N/A

### Code of Conduct & Contributing Guidelines 

By submitting creating this pull request, you agree to follow our [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md):
- [X] I agree to follow the [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: UtahModules/Sources/UtahTrends/DataCard.swift [hunk 1]
```diff
@@ -54,11 +58,28 @@ struct DataCard: View {
...
         .task {
-            await firestoreManager.loadObservations(metricCode: "55423-8")
-            recalculateChartData(basedon: firestoreManager.observations)
+            if title == "Daily Step Count" {
+                await firestoreManager.loadObservations(metricCode: "55423-8")
+                recalculateChartData(basedon: firestoreManager.observations)
+            } else if title == "Edmonton Frail Scale" {
+                await firestoreManager.loadSurveys()
+                getSurveyData(surveyType: "edmonton")
+            } else if title == "Veines Survey Score" {
```
Comment (jenlmoore) [inline UtahModules/Sources/UtahTrends/DataCard.swift:68]: this should also have a case for 'wiq' right?

Comment (jenlmoore) [reply UtahModules/Sources/UtahTrends/DataCard.swift:68]: we have 3 surveys

Comment (jennxu23) [reply UtahModules/Sources/UtahTrends/DataCard.swift:68]: I just changed this in a separate PR!

**Comment by jenlmoore** [comment]
I just want to confirm that with this PR, all surveys will now have corresponding charts appear when the data card is clicked right? There seemed to be no WIQ data card before so want to make sure.

**Comment by jennxu23** [comment]
>

File: UtahModules/Sources/UtahTrends/DataCard.swift (hunk 1 above)
Comment (jennxu23) [reply UtahModules/Sources/UtahTrends/DataCard.swift:68]: ^ update: this change is in this PR

**Review by jenlmoore — CHANGES_REQUESTED** [review]
wait for my updates

**Review by jenlmoore — APPROVED** [review]
approving so we can get it on testflight asap.. should be good now w my edits

**Pull request merged**